test_scripts/Polices/build_options/ATF_PTU_Trigger_IGN_cycles.lua

```diff
@@ -16,15 +16,17 @@
 
 --[[ General configuration parameters ]]
 config.deviceMAC = "12ca17b49af2289436f303e0166030a21e525d266e209267433801a8fd4071a0"
```

Comment:
@dboltovskyi, please update appHMIType to different than NAVIGATION, to be sure that this will not trigger additional PTU.